.github/workflows/coverage-check.yml

```diff
@@ -53,6 +53,7 @@ jobs:
                   "*scripts*" \
                   "src/dollar/mocks/*" \
                   "src/dollar/utils/*" \
+                  "src/old/*" \
```

Comment:
Perhaps this folder could be renamed to 'deprecated', meaning still functional but will be replaced?

Reply:
yeah and name something like "Artifacts"

Reply:
Renamed `old` to `deprecated` https://github.com/ubiquity/ubiquity-dollar/pull/842/commits/261eabed1099a8dbe4df56373be7068f254168d0